lib/config/migrations/custom/host-rules-migration.ts

```diff
@@ -0,0 +1,30 @@
+import type { HostRule } from '../../../types';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class HostRulesMigration extends AbstractMigration {
+  override readonly propertyName = 'hostRules';
+
+  override run(value: Record<string, unknown>[]): void {
+    const newHostRules: HostRule[] = value.map((rule) => {
+      const newRule = { ...rule };
+      newRule.hostType ??= newRule.platform;
+      newRule.matchHost ??=
```

Comment:
This will cause a different field order? 🤔

I thought this should not happen.

Reply:
Yeah, you are right, fixed